definitions/defaults/midi.lua

```diff
@@ -25,10 +25,10 @@ return {
   command = {
     ["n"] = "AddNextNoteToSelection",
     ["N"] = "AddPrevNoteToSelection",
-    ["+"] = "MidiZoomInVert",
```

Comment:
I changed so that the zoom x/y axis are the same for main and midi.

Reply:
okay